### PR TITLE
Run dotnet format

### PR DESCRIPTION
**Pull request by Hinton**
## Type of change
- [ ] Bug fix
- [ ] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective

This PR applies the dotnet format tool on the codebase. It's a follow up to https://github.com/bitwarden/server/pull/1756.

Any existing branch will have a fair bit of merge conflicts after this change. So I've prepared some documentation on how to easily get in-sync again.

1. Check out your local Branch
2. Run `git merge <hash config>`
3. Resolve any merge conflicts, commit.
4. Run `dotnet tool run dotnet-format`
5. Commit
6. Run `git merge -Xours <hash this PR>`
7. Push

## Testing requirements

N/A, no functional changs.

## Before you submit
- [ ] If making database changes - I have also updated Entity Framework queries and/or migrations
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

File: test/Common/AutoFixture/FixtureExtensions.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-using AutoFixture;
+﻿using AutoFixture;
```
Comment (cscharf) [inline test/Common/AutoFixture/FixtureExtensions.cs]: What actually changed here? I have no idea/can't tell.

File: test/Core.Test/Repositories/EntityFramework/EqualityComparers/InstallationCompare.cs [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 namespace Bit.Core.Test.Repositories.EntityFramework.EqualityComparers
 {
-    public class InstallationCompare: IEqualityComparer<Installation>
+    public class InstallationCompare : IEqualityComparer<Installation>
     {
         public bool Equals(Installation x, Installation y)
         {
-            return  x.Email == y.Email &&
+            return x.Email == y.Email &&
             x.Key == y.Key &&
```
Comment (cscharf) [inline test/Core.Test/Repositories/EntityFramework/EqualityComparers/InstallationCompare.cs]: I can't say I'm fond of not indenting `return` statements when they go to the next line, but at least it's in a block I guess.

**Review by cscharf — APPROVED** [review]
Looks good, just a few thoughts and generic feedback in general, not that there's anything to fix or do.

[x] verified there are only formatting changes, and formatting changes look good
[x] verified there are no logic changes or shenanigans going on, and there's not, which is good!

File: test/Core.Test/Repositories/EntityFramework/EqualityComparers/InstallationCompare.cs (hunk 2 above)
Comment (Hinton) [reply test/Core.Test/Repositories/EntityFramework/EqualityComparers/InstallationCompare.cs]: Yea ... it seems dotnet format doesn't handle formatting for these scenarios. I'm not going to manually change it though and will be something we can touch upon next time we edit that file.

File: test/Common/AutoFixture/FixtureExtensions.cs (hunk 1 above)
Comment (Hinton) [reply test/Common/AutoFixture/FixtureExtensions.cs]: Probably.

**Review by cscharf — APPROVED** [review]

**Pull request merged**